src/components/Home.jsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 import Navigation from './Navigation'
 import SearchBar from './SearchBar'
+import DiplayBooks from './DiplayBooks'
 
 const Home = () => {
   return (
     <div>
         <Navigation/>
-        <SearchBar/>
```

Comment:
just to double check, was SearchBar intended to be removed here?

Reply:
@jnorrick  I am currently not using SearchBar component so, I removed it in Home.jsx.